src/material-examples/chips-autocomplete/chips-autocomplete-example.ts

```diff
@@ -0,0 +1,83 @@
+import {Component, ViewChild, ElementRef} from '@angular/core';
+import {MatChipInputEvent, MatAutocompleteSelectedEvent} from '@angular/material';
+import {ENTER} from '@angular/cdk/keycodes';
+import {FormControl} from '@angular/forms';
+import {Observable} from 'rxjs/Observable';
+import 'rxjs/add/operator/startWith';
+import 'rxjs/add/operator/map';
+
+const COMMA = 188;
```

Comment:
I wonder if we should just add this to cdk/keycodes? and also semicolon if its not there already

Reply:
It wasn't at the time of this PR but it is now :)